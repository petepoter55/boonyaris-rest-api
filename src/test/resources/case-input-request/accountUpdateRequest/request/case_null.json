{
  "id" : 2,
  "username": null,
  "role": null,
  "firstname": "บูญฤทธิ์",
  "lastname": "เพ็ญสิริ",
  "email": "boonyaris2@gmail.com",
  "createBy": "usperadmin",
  "delFlag": null
}