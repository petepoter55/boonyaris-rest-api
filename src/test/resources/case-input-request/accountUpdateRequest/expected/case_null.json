[
  "username: must not be null"
]